### PR TITLE
OTEL revamp rough draft

### PR DESCRIPTION
**Pull request by lukebarrett24**

### CODE REVIEW

File: data-ingestion-apis/opentelemetry-metrics.mdx [hunk 1]
```diff
@@ -1,42 +1,77 @@
...
-|k8s.pod | k8s.pod.uid |
-| k8s.deployment | k8s.deployment.uid |
-| k8s.daemonset | k8s.daemonset.uid |
-| k8s.replicaset| k8s.replicaset.uid |
-| k8s.statefulset | k8s.statefulset.uid |
-| k8s.namespace | k8s.namespace.uid |
-| service | service.name |
-| os | os.type |
-| custom | custom |
+### Step 1: Setup Resource Attributes
```
Comment (Zamichel) [inline data-ingestion-apis/opentelemetry-metrics.mdx:15]: Agent is not required, however there is an extra config that needs to be applied if you're bypassing/not using the agent. That config is `target`, ed has details on how to apply it.

File: data-ingestion-apis/opentelemetry-metrics.mdx [hunk 2]
```diff
@@ -1,42 +1,77 @@
 ---
-title: "Data Ingestions API"
+title: "Customizing Telemetry Ingestion"
+sidebarTitle: "OpenTelemetry"
 ---
 
 # Overview
 This guide provides an overview and setup instructions for Middleware’s OpenTelemetry Data Ingestion API. The OpenTelemetry Ingestion API has two endpoints: [metrics](#metrics-endpoint) and [logs](#logs-endpoint).
```
Comment (Zamichel) [inline data-ingestion-apis/opentelemetry-metrics.mdx:7]: We recommend using Middleware's Agent and associated SDKs to ingest data, however for more advanced users we do allow using the OpenTelemetry Protocol Exporter. Each language will have a different approach which can be an adaptation from our relevant Host Agent.

The example below is using javascript and an abstraction from our [node agent](https://github.com/middleware-labs/agent-apm-nodejs). All other agents can be found in our [middleware-labs repo](https://github.com/middleware-labs). If you need assistance with a specific language we are happy to work with you, however we do not have every approach documented at this time.

File: data-ingestion-apis/opentelemetry-metrics.mdx [hunk 3]
```diff
@@ -1,42 +1,86 @@
...
-| os | os.type |
-| custom | custom |
+# Installation
 
-## Metrics Endpoint
+### Step 1: Setup Resource Attributes
 
-The Metrics endpoint lets you send custom metrics to the Middleware backend. 
+Embed OpenTelemetry Resource Attributes:
+<Note> Learn more about OpenTelemetry resources here </Note>
```
Comment (Zamichel) [inline data-ingestion-apis/opentelemetry-metrics.mdx:20]: Missing link

File: data-ingestion-apis/opentelemetry-metrics.mdx [hunk 4]
```diff
@@ -1,42 +1,86 @@
...
-| service | service.name |
-| os | os.type |
-| custom | custom |
+# Installation
 
-## Metrics Endpoint
+### Step 1: Setup Resource Attributes
 
-The Metrics endpoint lets you send custom metrics to the Middleware backend. 
+Embed OpenTelemetry Resource Attributes:
```
Comment (Zamichel) [inline data-ingestion-apis/opentelemetry-metrics.mdx:19]: Let's give a "why we're talking about this for this section" e.g. "Middleware connects via Otel using the mw.account_key which can be found [in the x section of the docs](with some link). To get started, you need to set up appropriate Resource attributes, seen below:"

File: data-ingestion-apis/opentelemetry-metrics.mdx [hunk 5]
```diff
@@ -1,42 +1,86 @@
...
+  [SemanticResourceAttributes.SERVICE_NAME]: "{your-service-name}",
+  ["mw_agent"]: true,
+  ["mw.account_key"]: "{your-api-key}",
+});
+```
+### Resource Attributes
+
+| Field | Type | Required | description |
+| :--: |:--:| :---: | :----:| 
+| mw.account_key | string | ✅ | API key that authenticates your MW Account |
```
Comment (Zamichel) [inline data-ingestion-apis/opentelemetry-metrics.mdx:33]: For mw_agent, make the description: "true if using a host agent, false if running serverless"

File: data-ingestion-apis/opentelemetry-metrics.mdx [hunk 6]
```diff
@@ -1,42 +1,86 @@
...
+| mw_agent | boolean |✅ | Required by MW |
+| SERVICE_NAME | string | ✅ | Logical name of service required by OpenTelemetry |
```
Comment (Zamichel) [inline data-ingestion-apis/opentelemetry-metrics.mdx:35]: Change descriptiong to "Logical name of the service, and desired filter value in Middleware"

File: data-ingestion-apis/opentelemetry-metrics.mdx [hunk 7]
```diff
@@ -93,139 +137,9 @@ curl -X POST "https://demo.middleware.io/v1/metrics" \
...
-|||
-| :-------------: | :--------: |
-| 1-4 | TRACE |
-| 5-8 | DEBUG |
-| 9-12 | INFO |
-| 13-16 | WARN |
-| 17-20 | ERROR |
-| 21-24 | FATAL |
+### Step 3: Send Data to Middleware
+Once you have setup your resource attributes, initialize your providers, and configure your exporters, begin sending data to Middleware.
```
Comment (Zamichel) [inline data-ingestion-apis/opentelemetry-metrics.mdx:143]: Let's add an example of how each of those tracers may be used. I know Ed has some somewhere that can be asily plopped in.

"Once you have setup....some example calls using {w/e language ed is using} is below"

Comment (lukebarrett24) [reply data-ingestion-apis/opentelemetry-metrics.mdx:144]: I will add this to the guides section

**Pull request closed without merging**